packages/aws-cdk-lib/aws-events-targets/test/ecs/event-rule-target.test.ts

```diff
@@ -1095,3 +1096,100 @@ test.each([
     ],
   });
 });
+
+test('Non-imported TaskDefinition role is targeting by Ref', () => {
+  const taskDefinition = new ecs.FargateTaskDefinition(stack, 'TaskDef');
+  taskDefinition.addContainer('TheContainer', {
+    image: ecs.ContainerImage.fromRegistry('henk'),
+  });
+
+  const rule = new events.Rule(stack, 'Rule', {
+    schedule: events.Schedule.rate(cdk.Duration.hours(1)),
+  });
+
+  rule.addTarget(
+    new EcsTask({
+      cluster: cluster,
+      taskDefinition: taskDefinition,
+    }),
+  );
+
+  const policyMatch = Match.objectLike({
+    PolicyDocument: {
+      Statement: Match.arrayWith([
+        Match.objectLike({
+          Action: 'ecs:RunTask',
+          Resource: {
+            Ref: 'TaskDef54694570',
```

Comment:
Does this policy have the trailing `:*` ?

Reply:
This uses a CloudFormation (CFN for short) `Ref` token to reference the TaskDefinition resource. According to the [CFN doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-taskdefinition.html#aws-resource-ecs-taskdefinition-return-values), the token will resolve to the TaskDefinition ARN >with< the revision number (but not `*`).

Reply:
👍🏼  – I wanted to make sure there was either a revision number or a `*` at the end.